pkg/controller/persistentvolume/persistentvolume_provisioner_controller.go

```diff
@@ -330,15 +330,19 @@ func (controller *PersistentVolumeProvisionerController) Stop() {
 	}
 }
 
-func newProvisioner(plugin volume.ProvisionableVolumePlugin, claim *api.PersistentVolumeClaim) (volume.Provisioner, error) {
+func newProvisioner(plugin volume.ProvisionableVolumePlugin, claim *api.PersistentVolumeClaim, pv *api.PersistentVolume) (volume.Provisioner, error) {
+	tags := make(map[string]string)
+	tags[cloudVolumeCreatedForClaimNamespaceTag] = claim.Namespace
+	tags[cloudVolumeCreatedForClaimNameTag] = claim.Name
+	if pv != nil {
```

Comment:
`newProvisioner` is called twice and at the first call the PV is not yet availabe, it's created by the returned provisioner.

https://github.com/kubernetes/kubernetes/pull/18957/files#diff-fc923807bec23504369a29c0f94b6055R175